### PR TITLE
Update the package information

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

File: package/skelcd-control-MicroOS.changes [hunk 1]
```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Mon Jun 28 12:39:32 UTC 2021 - Alberto Planas Dominguez <aplanas@suse.com>
+
+- Add remote attestation roles: micro_os_role_ra_{agent,verifier}
+
```
Comment (jreidinger) [inline package/skelcd-control-MicroOS.changes:6]: convention is to add version below, so it is easier to see which version cause which change.

```suggestion
- 20210628

```

Comment (aplanas) [reply package/skelcd-control-MicroOS.changes:6]: Done.

**Review by jreidinger — APPROVED** [review]

**Comment by jreidinger** [comment]
thanks for patch

**Comment by jreidinger** [comment]
Ah, one more think missing ( I forgot it sorry ). It needs issue ID either bugzilla or jira entry who requested such change.

**Comment by aplanas** [comment]
> Stopping, missing new bugzilla or fate entry in the *.changes file.
> e.g. bnc#<number> or fate#<number>

Do I need to create a bsc for a feature?

**Comment by jreidinger** [comment]
> > Stopping, missing new bugzilla or fate entry in the *.changes file.
> > e.g. bnc# or fate#
> 
> Do I need to create a bsc for a feature?

well, if it is feature and has already jira entry you are fine, we even accept old fate numbers...but if there are no such entries, then creating bsc is probably the easiest way

**Comment by aplanas** [comment]
>  creating bsc is probably the easiest way

I see, thanks. Done here: https://github.com/yast/skelcd-control-MicroOS/pull/41

**Pull request merged**